ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -846,8 +847,8 @@ end
 ```
 
 - [ ] `@user.user_city`
-- [ ] `@user.location_city`
-- [x] `@user.city`
+- [x] `@user.location_city`
+- [ ] `@user.city`
```

Comment:
I was unable to find a documentated rails version that implied that `@user.city` would ever be the right answer. Unless there was a point in which `prefix: true` didn't exist, `@user.location_city` is, in all cases, the right answer.